modules/test/playwright/tests/message-boards-web/message-boards-web.spec.ts

```diff
@@ -48,3 +49,35 @@ test('LPD-25630 Show the status to guest user', async ({
 
 	await expect(page.getByText('Pending')).toBeVisible();
 });
+
+test('LPD-27633 Do not show site in breadcrumb', async ({
+	messageBoardsPage,
+	messageBoardsWidgetPage,
+	page,
+	site,
+	workflowPage,
+}) => {
+	await page.goto(site.friendlyUrlPath);
+
+	const layout = await messageBoardsWidgetPage.addMessageBoardsPortlet(site);
+
+	const categoryName = getRandomString();
+
+	await messageBoardsWidgetPage.addCategory(site, layout, categoryName);
+
+	await page
+		.locator(
+			'[id="_com_liferay_message_boards_web_portlet_MBPortlet_mbCategoriesSearchContainer_1_menu"]'
+		)
+		.click();
```

Comment:
This instruction was randomly failing. I added a `waitFor` and it seems to work better.

```javascript
	const searchMenu = page
		.locator(
			'[id="_com_liferay_message_boards_web_portlet_MBPortlet_mbCategoriesSearchContainer_1_menu"]'
		);

	await searchMenu.waitFor();
	await searchMenu.click();
```

Reply:
Thank you Marco, I've made those changes and I'll run the tests again to see if it helped.